README.md

```diff
@@ -1,4 +1,4 @@
-# pattern-library
+## DGL 104 Pattern Library
```

Comment:
Can you comment on the reason why you have used a second level heading (i.e. ##) instead of a first level heading (i.e. #)?

Reply:
Oops! Just a mistake! Let me fix.